src/scripts/clipperUI/regionSelector.tsx

```diff
@@ -199,8 +200,17 @@ class RegionSelectorClass extends ComponentBase<RegionSelectorState, ClipperStat
 	 * the maximum allowed size)
 	 */
 	private startRegionClip() {
+		// Taken from https://www.kirupa.com/html5/detecting_retina_high_dpi.htm
+		// We check this here so that we can log it as a custom property on the regionSelectionProcessingEvent
+		const query = "(-webkit-min-device-pixel-ratio: 2), (min-device-pixel-ratio: 2), (min-resolution: 192dpi)";
+		const isHighDpiScreen = matchMedia(query).matches;
+		const isFirefoxWithHighDpiDisplay = this.props.clipperState.clientInfo.clipperType === ClientType.FirefoxExtension && isHighDpiScreen;
+
 		// Firefox reports this value incorrectly if this iframe is hidden, so store it now since we know we're visible
-		this.devicePixelRatio = window.devicePixelRatio;
+		// In addition to this, Firefox currently has a bug where they are not using devicePixelRatio correctly 
+		// on HighDPI screens such as Retina screens or the Surface Pro 4
+		// Bug link: https://bugzilla.mozilla.org/show_bug.cgi?id=1278507 
+		this.devicePixelRatio = isFirefoxWithHighDpiDisplay ? window.devicePixelRatio / 2 : window.devicePixelRatio;
```

Comment:
Yes, it will be.

I hardcoded the first and second points used in the region capture and Firefox and Chrome returned different results, which seems the bug is in the implementation, and there doesn't seem to be a way to collapse the logic to use the "correct" values once FF fixes the bug.